dashboard/app/helpers/levels_helper.rb

```diff
@@ -403,9 +371,9 @@ def blockly_options
     app_options[:firebaseAuthToken] = firebase_auth_token if @game == Game.applab
     app_options[:isAdmin] = true if @game == Game.applab && current_user && current_user.admin?
     app_options[:isSignedIn] = !current_user.nil?
-    app_options[:pinWorkspaceToBottom] = true if enable_scrolling?
-    app_options[:hasVerticalScrollbars] = true if enable_scrolling?
-    app_options[:showExampleTestButtons] = true if enable_examples?
+    app_options[:pinWorkspaceToBottom] = true if l.enable_scrolling?
+    app_options[:hasVerticalScrollbars] = true if l.enable_scrolling?
```

Comment:
(Unrelated to this PR) Why have one option when we can have two that mean the same thing? 😂